src/CalendarUtils.php

```diff
@@ -625,7 +625,7 @@ private static function getDayNames($day, $shorten = false, $len = 1, $numeric =
         return ($numeric) ? $n : (($shorten) ? mb_substr($ret, 0, $len, 'UTF-8') : $ret);
     }
 
-    private static function getMonthNames($month, $shorten = false, $len = 3)
```

Comment:
Don't change the access modifier. please add new public function if you need

Reply:
Alright, its reverted now and i will raise a new pull request for adding new public function.
Subject of this request has been changed.